website/source/downloads.html.erb

```diff
@@ -9,40 +9,51 @@ description: |-
 <h1>Download Nomad</h1>
 
 <section class="downloads">
-	<div class="description row">
-		<div class="col-md-12">
-			<p>
-			Below are all available downloads for the latest version of Nomad
-			(<%= latest_version %>). Please download the proper package for your
-			operating system and architecture. You can find SHA256 checksums
-			for packages <a href="https://dl.bintray.com/mitchellh/nomad/nomad_<%= latest_version %>_SHA256SUMS?direct">here</a>. You can verify the SHA256 checksums using <a href="https://hashicorp.com/security.html">our PGP public key</a> and the <a href="https://dl.bintray.com/mitchellh/nomad/nomad_<%= latest_version %>_SHA256SUMS.sig?direct">SHA256SUMs signature file</a>.
-			</p>
-		</div>
-	</div>
-	<% product_versions.each do |os, versions| %>
-		<div class="row">
-		<div class="col-md-12 download">
-			<div class="icon pull-left"><%= system_icon(os) %>
-		</div>
-			<div class="details">
-				<h2 class="os-name"><%= os %></h2>
-				<ul>
-					<% versions.each do |url| %>
-						<li><a href="<%= url %>"><%= arch_for_filename(url) %></a></li>
-					<% end %>
-				</ul>
-				<div class="clearfix">
-				</div>
-			</div>
-		</div>
-	</div>
-<% end %>
+  <div class="description row">
+    <div class="col-md-12">
+      <p>
+        Below are the available downloads for the latest version of Nomad
+        (<%= latest_version %>). Please download the proper package for your
+        operating system and architecture.
+      </p>
+      <p>
+        You can find the
+        <a href="https://releases.hashicorp.com/nomad/<%= latest_version %>/nomad_<%= latest_version %>_SHA256SUMS">
+          SHA256 checksums for Nomad <%= latest_version %>
+        </a>
+        online and you can
```

Comment:
It's an SEO thing. We want the link target to read "SHA256 checksums for Nomad". If you create an a link with just "here", it hurts your searchability.